src/style.css

```diff
@@ -1 +1,24 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
```

Comment:
Following the checklist, avoid using tag names for styling. It's better to use classes to apply styles to avoid issues with specificity and potential conflicts in larger stylesheets.